### ISSUE TITLE
Server Error when trying to access Configuration Settings

### ISSUE THREAD

**Issue by ringtailedfox** [ZeroNetX/ZeroNet#164]
### Step 1: Please describe your environment

  * ZeroNet version: 0.7.9
  * Operating system: Microsoft Windows 10 Professional 64-bit Edition 21H2 (build 19044.1682)
  * Web browser: Microsoft Edge 101.0.1210.32 (Official build) (64-bit)
  * Tor status: not available/always/disabled
  * Opened port: yes/no
  * Special configuration: No

### Step 2: Describe the problem:
Cannot edit configuration settings via UI

#### Steps to reproduce:

  1. Start ZeroNet
  2. Click on the link to Configuration
  3. Read the following error below:
  
Server error
Err: AttributeError: 'NoneType' object has no attribute 'content_manager' in <core>/src/Ui/UiServer.py line 108 > ...

#### Observed Results:

  * What happened? This could be a screenshot, a description, log output (you can send log/debug.log file to hello@zeronet.io if necessary), etc.

#### Expected Results:

  * What did you expect to happen?
I expected to be able to load the configuration screen.

**Comment by slrslr** [comment]
Yes, it happened also in latest Zeronet Version: 0.8.5 r4625 on stable Linux Debian:

`Ui.UiServer Error 500: Err: AttributeError: 'NoneType' object has no attribute 'content_manager' in UiServer.py line 108 > UiRequest.py line 192 > UiPluginManagerPlugin.py line 33 > UiFileManagerPlugin.py line 21 > UiConfigPlugin.py line 29 > UiRequest.py line 524`

but after a minute, i was able to load /Config without error.
